### PR TITLE
address colored grid borders better (bug 1130989)

### PR DESCRIPTION
**Pull request by spasovski**

### CODE REVIEW

File: src/media/css/app-list.styl [hunk 1]
```diff
@@ -7,6 +7,7 @@ App listing.
 @import 'lib';
 
 $border = 1px solid $tile-border;
+$colored-grid-border = 1px solid rgba($white, 0.4);
```
Comment (ngokevin) [inline src/media/css/app-list.styl]: bit of a misnomer :)

File: src/media/css/app-list.styl [hunk 2]
```diff
@@ -129,6 +130,20 @@ $desktop-tile-padding = 20px 50px;
             padding-top: 30px;
         }
     }
+    // Colored grids.
+    .app-list-colored {
+        .app-list-colored:after {
```
Comment (ngokevin) [inline src/media/css/app-list.styl]: does this work? ```.app-list-colored .app-list-colored:after```

File: src/media/css/app-list.styl (hunk 1 above)
Comment (ngokevin) [inline src/media/css/app-list.styl]: I'd put this in colors.styl

**Comment by ngokevin** [comment]
The current grid code is required? No need for deletions?

File: src/media/css/app-list.styl (hunk 2 above)
Comment (spasovski) [inline src/media/css/app-list.styl]: Nope, copy-pasta when it lived in feed/colors.styl. The one there still works this one is not needed.

**Comment by spasovski** [comment]
No deletions since we need the existing rules for non-colored app lists. New/Popular/Search etc. These are purely border overrides...which I am fine with since it's only one level.

**Comment by ngokevin** [comment]
non-colored app lists have different grid than colored?

**Comment by spasovski** [comment]
Different border color and different colored pseudo-element. Everything else is the same.

**Comment by ngokevin** [comment]
What about the border-right: 0? That's not about colors, should that be pulled into the base grid?

**Comment by spasovski** [comment]
Nope, that's to reset the previously-set `.app-list-app {border...}` rules. The regular grids already have that reset and this won't affect them since they lack the parent class.

**Comment by ngokevin** [comment]
I'm good with it. Let's have @ddurst sign off too maybe

**Comment by ddurst** [comment]
r+

**Comment by ngokevin** [comment]
nice fix :)

**Pull request merged**